### DISCUSSION TITLE
[SUGGESTION] Build folders

### DISCUSSION THREAD

**Discussion by Briaireous** [tsunamayo/Starship-EVO#4022] [Suggestions]
Greetings Tsuna, I do not know about how anyone else feels about the need to search around awhile to find and spawn an entity when interacting with the new entity blocks, but I find it a bit annoying after having quite a few subscriptions and builds in that menu. 

So, my suggestion would be for there to be an option of make folders in that menu so we can better organize our builds. Also would be nice if the all workshop subscriptions can get automatically put into a folder for the subs in that menu.

This suggestion is just a quality of life thing I've been thinking about. Probs influenced by my approach to my newest build project.

**Comment by matejkar** [comment]
I definetly agree, i don't think its hard to add, and would definetly help

**Comment by daOyster** [comment]
I agree, this would be a nice quality of life addition to have.

**Comment by tsunamayo** [comment]
Yep this already have been suggested. It is not hard to do, just time consuming...

**Comment by Briaireous** [comment]
Ah, sorry about that.
